tests/test_core.py

```diff
@@ -999,6 +1001,48 @@ def test_objcinstance_nspoint_repr(self):
         self.assertEqual(repr(my_point), "<NSPoint(10.0, 20.0)>")
         self.assertEqual(str(my_point), "(10.0, 20.0)")
 
+    def test_objcinstance_nsrect_repr(self):
+        """Test NSRect repr and str returns correct value."""
+
+        my_rect = NSRect(NSPoint(10, 20), NSSize(5, 15))
+        self.assertEqual(repr(my_rect), "<NSRect(5.0x15.0 @ 10.0, 20.0)>")
+        self.assertEqual(str(my_rect), "(5.0x15.0 @ 10.0, 20.0)")
+
+    def test_objcinstance_nssize_repr(self):
+        """Test NSSize repr and str returns correct value."""
+
+        my_size = NSSize(5, 5)
```

Comment:
When writing a test that has multiple inputs, it's generally a good idea to use different values - that way, you can catch accidental transposition errors. For example, if the implementation for this method printed the width for the height, and the height for the width, this test wouldn't identify the problem.

Reply:
The same applies to the following tests for range, insets etc.